src/util/localization.ts

```diff
@@ -4,158 +4,158 @@ export interface localizeString{
   schinese : string,
   tchinese : string,
   english : string,
-  german:string,
+  german : string,
 }
 var localizeMap:Map<number,localizeString>= new Map([
   [1,{
     schinese:"设置",
     tchinese:"設定",
     english:"SETTINGS",
```

Comment:
Should this be ALL CAPS?
```suggestion
    english:"Settings",
```